presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/PlanMatchPattern.java

```diff
@@ -699,4 +705,49 @@ private static String indentString(int indent)
     {
         return Strings.repeat("    ", indent);
     }
+
+    public static class Ordering
+    {
+        private final String field;
+        private final SortItem.Ordering sortOrder;
+        private final SortItem.NullOrdering nullOrder;
+
+        public Ordering(String field, SortItem.Ordering sortOrder, SortItem.NullOrdering nullOrder)
+        {
+            this.field = field;
+            this.sortOrder = sortOrder;
+            this.nullOrder = nullOrder;
+        }
+
+        public String getField()
+        {
+            return field;
+        }
+
+        public SortItem.Ordering getSortOrder()
+        {
+            return sortOrder;
+        }
+
+        public SortItem.NullOrdering getNullOrder()
+        {
+            return nullOrder;
+        }
+
+        @Override
+        public String toString()
+        {
+            String result = field + " " + sortOrder;
+            if (nullOrder != SortItem.NullOrdering.UNDEFINED) {
+                result += " NULLS " + nullOrder;
+            }
+
+            return result;
+        }
+    }
+
+    public static Ordering sort(String field, SortItem.Ordering sortOrder, SortItem.NullOrdering nullOrder)
```

Comment:
I thought we usually put static things at the top of the class, especially when they are public factory methods

Reply:
I'll move it closer to where all the other static methods are